kura/org.eclipse.kura.rest.security.provider/src/main/java/org/eclipse/kura/internal/rest/security/provider/SecurityRestService.java

```diff
@@ -98,19 +96,57 @@ public Response reloadSecurityPolicyFingerprint() {
     }
 
     /**
-     * POST method
-     * <br />
-     * This method allows the reload of the command line fingerprint
-     * 
+     * POST method <br /> This method allows the reload of the command line fingerprint
      */
-
     @POST
     @RolesAllowed(REST_ROLE_NAME)
     @Path("/command-line-fingerprint/reload")
     @Produces(MediaType.APPLICATION_JSON)
     public Response reloadCommandLineFingerprint() {
         try {
-            logger.debug(DEBUG_MESSSAGE, "reloadCommandLineFingerprint");
+            logger.debug(DEBUG_MESSAGE, "reloadCommandLineFingerprint");
+            this.security.reloadCommandLineFingerprint();
+        } catch (Exception e) {
+            throw DefaultExceptionHandler.toWebApplicationException(e);
+        }
+
+        return Response.ok().build();
+    }
+
+    /**
+     * POST method <br /> This method replaces the security policy with the default production one. Then a fingerprint
+     * reload is performed.
+     */
+    @POST
+    @RolesAllowed(REST_ROLE_NAME)
+    @Path("/security-policy/apply-default-production")
+    public Response applyDefaultProductionSecurityPolicy() {
```

Comment:
Please create a security/v2 REST endpoint and SEC-V2 request handler that provides the new methods and the existing leaving SEC-V1 in place unmodified